docs/conf.py

```diff
@@ -110,7 +110,6 @@
 # directories to ignore when looking for source files.
 exclude_patterns = [
     "_build",
-    "**/.nox/**/*",
```

Comment:
I needed this line to build the docs locally. Is it breaking something with CI?

Reply:
No, it seems that service accounts keys for Python CI have expired yesterday, there are similar failures in other repos, and even in some non-Python ones.

Reply:
Yay, samples tests now pass. :tada: